client/libs/material/src/lib/sermon-table/sermon-table.component.scss

```diff
@@ -0,0 +1,47 @@
+@import 'libs/theme/variables.scss';
+@import 'libs/theme/main.scss';
+
+.full-width-table {
+  width: 100%;
+}
+
+.bg-color-primary {
```

Comment:
add this somewhere global, please. It might be good if you added all other `bg-*` classes based on our color palette in the variables file.